apps/designer/app/shared/css-utils/generate-css-text.ts

```diff
@@ -0,0 +1,44 @@
+import { json } from "@remix-run/node";
+import { db } from "@webstudio-is/project/server";
+import { utils } from "@webstudio-is/project";
+import { loadCanvasData } from "~/shared/db";
+import { createCssEngine } from "@webstudio-is/css-engine";
+import { idAttribute } from "@webstudio-is/react-sdk";
+import { addGlobalRules } from "~/canvas/shared/styles";
+import { BuildParams } from "../router-utils";
```

Comment:
use type hint always, we will automate the check through linter later

```suggestion
import type { BuildParams } from "../router-utils";
```